libc-test/build.rs

```diff
@@ -1378,6 +1378,8 @@ fn test_android(target: &str) {
             "SIG_DFL" | "SIG_ERR" | "SIG_IGN" => true, // sighandler_t weirdness
             // FIXME: still necessary?
             "SIGUNUSED" => true, // removed in glibc 2.26
+            // FIXME: remove once Android uses kernel headers >= 5.1 which expose this constant
+            "F_SEAL_FUTURE_WRITE" => true,
```

Comment:
I would prefer to just not expose this on Android for now.

Reply:
Done